ui/tests/integration/components/ldap/page/configure-test.js

```diff
@@ -0,0 +1,155 @@
+  });
+
+  test('it should render empty state when schema is not selected', async function (assert) {
+    await this.renderComponent();
+    // await this.pauseTest();
+    assert.dom('[data-test-empty-state-title]').hasText('Choose an option', 'Empty state title renders');
+    assert
+      .dom('[data-test-empty-state-message]')
+      .hasText('Pick an option above to see available configuration options', 'Empty state title renders');
+    assert.dom(selectors.save).isDisabled('Save button is disabled when schema is not selected');
+
+    await click(selectors.radioCard);
+    assert
+      .dom('[data-test-component="empty-state"]')
+      .doesNotExist('Empty state is hidden when schema is selected');
+  });
+
+  test('it should render validation messages for invalid form', async function (assert) {
+    await this.renderComponent();
+
+    await click(selectors.radioCard);
+    await click(selectors.save);
+
+    assert
+      .dom('[data-test-field="binddn"] [data-test-inline-error-message]')
+      .hasText('Administrator distinguished name is required.', 'Validation message renders for binddn');
+    assert
+      .dom('[data-test-field="bindpass"] [data-test-inline-error-message]')
+      .hasText('Administrator password is required.', 'Validation message renders for bindpass');
+    assert
+      .dom('[data-test-invalid-form-message] p')
+      .hasText('There are 2 errors with this form.', 'Invalid form message renders');
+  });
+
+  test('it should save new configuration without rotating root password', async function (assert) {
+    assert.expect(2);
+
+    this.server.post('/ldap-new/config', () => {
+      assert.ok(true, 'POST request made to save config');
+      return new Response(204, {});
+    });
+
+    await this.renderComponent();
+    await fillAndSubmit('without');
+
+    assert.ok(
+      this.transitionStub.calledWith('vault.cluster.secrets.backend.ldap.configuration'),
+      'Transitions to configuration route on save success'
+    );
+  });
+
+  test('it should save new configuration and rotate root password', async function (assert) {
+    assert.expect(3);
+
+    this.server.post('/ldap-new/config', () => {
+      assert.ok(true, 'POST request made to save config');
+      return new Response(204, {});
+    });
+    this.server.post('/ldap-new/rotate-root', () => {
+      assert.ok(true, 'POST request made to rotate root password');
+      return new Response(204, {});
+    });
+
+    await this.renderComponent();
+    await fillAndSubmit('with');
+
+    assert.ok(
+      this.transitionStub.calledWith('vault.cluster.secrets.backend.ldap.configuration'),
+      'Transitions to configuration route on save success'
+    );
+  });
+
+  test('it should populate fields when editing form', async function (assert) {
+    this.model = this.editModel;
+
+    await this.renderComponent();
+
+    assert
+      .dom('[data-test-radio-card="OpenLDAP"]')
```

Comment:
nit: can we use the `selectors.radioCard` here? :)

Reply:
Definitely. Good eye!